src/Sylius/Bundle/ProductBundle/Form/EventSubscriber/ProductOptionFieldSubscriber.php

```diff
@@ -59,7 +59,7 @@ public function preSetData(FormEvent $event)
         $form = $event->getForm();
 
         /** Options should be disabled for configurable product if it has at least one defined variant */
-        $disableOptions = (null !== $this->variantResolver->getVariant($product)) && (false === $product->hasVariants());
+        $disableOptions = (null !== $this->variantResolver->getVariant($product)) && $product->hasVariants();
```

Comment:
If product has variants the first condition (`(null !== $this->variantResolver->getVariant($product))` will always be true. It's redundant.